scrowl-io/scrowl
# [sc-207] - Set up React Redux Developer Tools

### Short summary

- Update docker-compose configuration to avoid creating a sub directory `scrowl-project` into the main `scrowl-project` directory;
- Install and set up the `electron-devtools-installer` module;
- Update the files structure to load a `.tsx` file that will render the actual React application.

---

### Test steps

1. Run `make start startOver=true` 
2. In the application window, check out for the React Devtools:

![Screen Shot 2022-02-10 at 3 17 31 PM](https://user-images.githubusercontent.com/98419841/153489674-e2c1750c-ccef-4688-9555-931ba84ac803.png)
